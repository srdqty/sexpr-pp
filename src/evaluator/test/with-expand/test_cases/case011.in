(let-syntax
(
  (stop (lambda (e) (car (cdr (stx-e e)))))
)
  (let-syntax
  (
    (ex (lambda (e) 
          (local-expand (car (cdr (stx-e e)))
                                  (list (syntax stop)))))
  )
    (ex (lambda (x) (let-syntax
                    (
                      (arg (lambda (e) (syntax (stop x))))
                    )
                    (arg))))))
