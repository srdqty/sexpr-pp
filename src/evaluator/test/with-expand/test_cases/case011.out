>> Error: top-level:13:54: undefined identifier: x
>> 
